recipes/jupyterlab-omnisci/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "jupyterlab-omnisci" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Quansight/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a6d30fbe6787b4ed4e2049ff298900fa5ab9919601f4ada0eb6a26ce9cf1ad6e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - jupyterlab
+    - pymapd
+    - ibis-framework
+    - pyyaml
+    - ipywidgets
+    - vdom
+    - altair
+    - vega_datasets
+
+test:
+  imports:
+    - jupyterlab_omnisci
+  commands:
+    - jupyter lab
+
+about:
+  home: https://github.com/Quansight/jupyterlab-omnisci
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'JupyterLab - Omnisci integration'
+
+  description: |
+    'JupyterLab - Omnisci integration'
+  dev_url: https://github.com/Quansight/jupyterlab-omnisci
+
+extra:
+  recipe-maintainers:
+    - OmniSci
```

Comment:
Is all of @OmniSci ok with this?

Reply:
Yes but I can add myself so that someone specific can be pinged when the recipe should be updated, etc.

Reply:
Who all is in pmnosci, so we can get their confirmation?

Reply:
Might be better to add a few of us explicitly:
- @andrewseidl 
- @randyzwitch 
- @mikehinchey
- @jpharvey

Reply:
Done!